### DISCUSSION TITLE
Server Registration Status update: Bad address Fix

### DISCUSSION THREAD

**Discussion by BentWrench** [jamulussoftware/jamulus#3438] [Bug Investigation]
Hello,

I have been running Jamulus server on a Raspberry Pi 4 for quite some time. Every time I rebooted it, the server would not register on the first try. I get the error "Server Registration Status update: Bad address". The server then waits 15 minutes, tries again, and is successful.

I figured out that the server was not properly waiting for the network to come online before starting, so I edited the default setting in the service file.

In the [Unit] section, I replaced
`After=network.target`

with

```
After=network-online.target
Wants=network-online.target
```

And that resolved the issue. I pulled that info from the excellent guide located here: [A collection of guides on using and configuring a Raspberry Pi](https://github.com/thagrol/Guides/blob/main/boot.pdf). Specifically, [boot.pdf](https://github.com/thagrol/Guides/blob/main/boot.pdf), which covers starting apps at boot time. It pulls info from [systemd.io](https://systemd.io/NETWORK_ONLINE/).

Wanted to share in case others have this issue, or you want to add this to the default service file.

Cheers!

**Comment by ann0see** [comment]
Thank you very much for reporting this. I'd assume that it depends a bit on which services run at which time during the boot process. 

It might make sense to open a pull request to apply the change here.

**Reply by rdica** [reply]
@ann0see I took the liberty of creating a PR to address the changes.
